src/main/java/org/ibp/api/java/impl/middleware/germplasm/validator/GermplasmNameRequestValidator.java

```diff
@@ -91,6 +91,14 @@ public void validateNameDeletable(final Integer gid, final Integer nameId) {
 
 	}
 
+	public void validateNameBelongsToGermplasm(final Integer gid, final Integer nameId) {
+		this.errors = new MapBindingResult(new HashMap<String, String>(), GermplasmNameRequestDto.class.getName());
+		this.germplasmValidator.validateGermplasmId(this.errors, gid);
```

Comment:
Should not be better to validate the germplasm before validating the name to make this function atomic and reusable? 
@lik-83